tests/functional/ext/typing/unnecessary_default_type_args.rc

```diff
@@ -1,3 +1,5 @@
 [main]
-py-version=3.10
 load-plugins=pylint.extensions.typing
+
+[testoptions]
+min_pyver=3.10
```

Comment:
Isn't py-version better than min_pyver ? (i.e. it means we can handle this on all interpreters). Or is there another reason to change this ?

Reply:
Didn't look to closely into `3.10` before so just assumed there was some syntax reason when there wasn't any. `collections.abc.Generator` is subscriptable since 3.9. Remove the testoption here.